test/Swapper.t.sol

```diff
@@ -0,0 +1,67 @@
+// SPDX-License-Identifier: AGPL-3.0-only
+pragma solidity ^0.8.19;
+
+import "forge-std/console2.sol";
+import "forge-std/Test.sol";
+
+import {Address} from "openzeppelin-contracts/utils/Address.sol";
+
+import {IwstETH} from "../src/interfaces/lido/IwstETH.sol";
+import {ILido} from "../src/interfaces/lido/ILido.sol";
+import {Constants as C} from "../src/lib/Constants.sol";
+
+import {Swapper} from "../src/steth/Swapper.sol";
+
+contract SwapperTest is Test {
+    using Address for address;
+
+    Swapper swapper;
+    IwstETH wstEth;
+
+    function setUp() public {
+        uint256 mainnetFork = vm.createFork(vm.envString("RPC_URL_MAINNET"));
+        vm.selectFork(mainnetFork);
+        vm.rollFork(18819890);
+
+        wstEth = IwstETH(C.WSTETH);
+        swapper = new Swapper();
+    }
+
+    function test_lidoSwapWethToWstEth_usesCurveForSmallAmounts() public {
+        uint256 wethAmount = 10 ether;
+        deal(C.WETH, address(this), wethAmount);
+
+        // expect a call to curvePool.exchange
+        vm.expectCall(address(swapper.curvePool()), abi.encodeCall(swapper.curvePool().exchange, (0, 1, wethAmount, 0)));
+
+        // execute the swap
+        bytes memory result = address(swapper).functionDelegateCall(
+            abi.encodeWithSelector(swapper.lidoSwapWethToWstEth.selector, wethAmount)
+        );
+
+        uint256 stEthAmountReceived = wstEth.unwrap(abi.decode(result, (uint256)));
```

Comment:
Can you also add an assertion that 

result == wstEth_balance

Reply:
[added](https://github.com/lindy-labs/sandclock-contracts/pull/152/commits/5b9a29cd838b8c76bd195686d1ee2ed464aabb7a)